src/node_api.cc

```diff
@@ -18,8 +18,6 @@
 #include "uv.h"
 #include "node_api.h"
 
-#define NAPI_VERSION  1
```

Comment:
I will submit this change upstream. I am checking with Michael [here](https://github.com/nodejs/node/pull/13207/files#r119427792) if he has strong reason for doing it this way.